deploy/scripts/install.sh

```diff
@@ -48,15 +49,33 @@ az feature register --namespace "Microsoft.ContainerService" --name "EnableWorkl
 # Captures the Azure cloud endpoints/suffixes
 az cloud show -o json > $PRJ_ROOT/deploy/cloud_endpoints.json
 
+AKS_VERSION=$(az aks get-versions -l $LOCATION | jq -r '.orchestrators[].orchestratorVersion' | sort -rV | head -n1)
```

Comment:
No need for `jq`. The orchestratorVersion can be pulled using an az cli query.
```suggestion
AKS_VERSION=$(az aks get-versions -l $LOCATION --query "orchestrators[].orchestratorVersion" -otsv | sort -rV | head -n1)
```
Thinking about it more, you may also not want to take the latest version since that's typically a preview version. Maybe you want the latest non-preview version from the list instead:
```suggestion
AKS_VERSION=$(az aks get-versions -l $LOCATION --query 'orchestrators[?!isPreview].orchestratorVersion' -otsv | sort -rV | head -n1)
```
I haven't verified, but I believe this is the version chosen if you just don't pass kubernetesVersion to the bicep `Microsoft.ContainerService/managedClusters` resource.

Reply:
Added the suggested change to the latest commit.